src/components/vlplotgroup/vlplotgroup.html

```diff
@@ -3,7 +3,7 @@
     class="vl-plot-group-header no-shrink">
     <div class="field-set-info" >
       <field-info ng-repeat="fieldDef in fieldSet"
-        ng-if="fieldSet"
```

Comment:
(I'm not sure how it works if `fieldSet` is `null`)

I guess the right fix should be `fieldSet && fieldDef.field`?